### PR TITLE
Create game

### PR DESCRIPTION
**Pull request by wettergrund**
Added functionality to create game + upload file

### CODE REVIEW

File: BlazorQuiz/Client/Pages/CreateQuizComponent.razor [hunk 1]
```diff
@@ -15,41 +15,77 @@
...
+        StateHasChanged();
+        // Now you can use the uploadedFileGuid as needed in this component
+    }
+
     private async Task CreateQuiz()
     {
-        var response = await _http.PostAsJsonAsync("api/PostQuiz", quizList);
+        string title = "Test";
+
+        var response = await _http.PostAsJsonAsync($"api/game/create/{title}", quizList);
```
Comment (berkowicz) [inline BlazorQuiz/Client/Pages/CreateQuizComponent.razor:72]: Shouldn't this be newgame since you updated the api call to newgame from create?

**Review by berkowicz — CHANGES_REQUESTED** [review]
See comment in code

File: BlazorQuiz/Client/Pages/CreateQuizComponent.razor (hunk 1 above)
Comment (wettergrund) [reply BlazorQuiz/Client/Pages/CreateQuizComponent.razor:72]: Renamed PostNewGame because of duplicated endpoints, but PostQuiz is the same still :)

**Review by berkowicz — APPROVED** [review]
LGTM :)

**Pull request merged**